test/user.test.js

```diff
@@ -1763,6 +1763,367 @@ describe('User', function() {
     });
   });
 
+  describe('Email Update', function() {
+    describe('User changing email property', function() {
+      var usersId;
+      var currentEmailCredentials = { email: 'm@c.com', password: 'bar' };
+      var updatedEmailCredentials = { email: 'n@d.com', password: 'bar' };
+
+      beforeEach(function(done) {
+        async.series([
+          function(next) {
+            User.create({ email: currentEmailCredentials.email,
+            password: currentEmailCredentials.password }, function(err, user) {
+              usersId = user.id;
```

Comment:
Also to avoid frequent `User.findById` in the tests below, it's better to store the full user `instance`.

```js
User.create(..., function(err, u) {
  user = u;
  userId = u.id; // probably not needed
  next();
});
```